test/expected-issue1037-output

```diff
@@ -0,0 +1,4 @@
+Root package issue1037-better-dependency-messages reference gitcompatibledubpackage 1.0.1 cannot be satisfied.
+Packages causing the conflict:
+	issue1037-better-dependency-messages depends on 1.0.1
+	b depends on ~>1.0.2
```

Comment:
For better clarity of the "test" folder, this should be named "issue1037-expected-output".

Reply:
but all the other 'expected output files' are also named this way, it would be weird to only name this one like that

Reply:
Missed that. I'll change those too then.